editor/icons/icon_refresh.svg

```diff
@@ -0,0 +1,125 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
The filename convention for icon files seems to be PascalCase. This can be easily changed

Reply:
Icon filenames have snake_case for the 3.2 branch. The should be changed while porting it to 4.0.

Reply:
@akien-mga  should I compress or remove unnecessary tags from SVG the file.

Reply:
See https://docs.godotengine.org/en/latest/development/editor/creating_icons.html#icon-optimization